packages/project/test/merge/workflow-generator.ts

```diff
@@ -0,0 +1,67 @@
+import { randomUUID } from 'node:crypto';
+
+class WorkflowGenerator {
+  ids = new Map<string, string>();
+  nodes: Record<string, any> = {};
+  constructor(def: string[], private name: string = 'workflow') {
+    for (const conn of def) {
+      const [from, to] = conn.split('-');
+      // create node for from and to
+      if (this.nodes[from]) {
+        if (to && !this.nodes[from].next[to])
+          this.nodes[from].next[to] = {
+            openfn: { id: this.uuid(`${from}-${to}`) },
```

Comment:
Notice how `this.uuid()` is always inside an `openfn` object?

Maybe this should just be `createNode(id, props)`, which does all this stuff automagically